monkestation/code/modules/antagonists/evil_clone/evil_clone_objective.dm

```diff
@@ -0,0 +1,42 @@
+/datum/objective/evil_clone/murder // The existance of the murderbone objective makes evil clones properly feared, so even when they aren't murderboning they will still be shunned and persecuted.
+	name = "clone supremacy"
+	explanation_text = "Kill everyone who isn't a clone of yourself. Do not spare the original."
```

Comment:
```suggestion
	explanation_text = "Make sure clones of yourself are the only ones alive. Do not spare the original."
```
hopefully this will reduce them just plasma flooding, although we might want to outright remove this